dashboard/client/src/pages/node/NodeRow.tsx

```diff
@@ -203,8 +203,14 @@ const WorkerRow = ({ node, worker }: WorkerRowProps) => {
       <TableCell align="center">N/A</TableCell>
       <TableCell>
         <Link to={workerLogUrl} target="_blank">
-          Log
-        </Link>
+          Logs
+        </Link><br/>
+        <a href={`/worker/traceback?pid=${pid}&ip=${ip}`} target="_blank" rel="noreferrer">
+            Stack&nbsp;Trace
+        </a><br/>
+        <a href={`/worker/cpu_profile?pid=${pid}&ip=${ip}&duration=5`} target="_blank" rel="noreferrer">
+            Flame&nbsp;Graph
+        </a><br/>
```

Comment:
As a follow-up, we should probably add some sort of hint or tooltip that says that this will begin profiling for the next 5 seconds after being clicked.

Reply:
Yeah--- I was thinking 5 seconds was short enough to be ok, but if we allow configuring longer durations then a dialog confirm would be useful.